strings/src/main/res/values-fi/strings.xml

```diff
@@ -941,8 +941,7 @@
   <!--Text for button that deletes all entities for the project-->
   <string name="clear_entities">Poista kaikki</string>
   <!--Text for button that adds a new entity list-->
```

Comment:
You should also get rid of the explanation text above.

Reply:
I've removed it for the string in `values`, but won't touch it for the translated files as that should happen automatically for when new translations are added.